src/views/Export/LeadsSelection/FilterForm/index.tsx

```diff
@@ -0,0 +1,547 @@
+    hasAssessment?: boolean;
+}
+
+const filterKeySelector = (d: FilterFields) => d.key;
+const optionTitleSelector = (d: BasicElement) => d.title;
+const optionIdSelector = (d: BasicElement) => d.id;
+const optionLabelSelector = (d: KeyValueElement) => d.value;
+const optionKeySelector = (d: KeyValueElement) => d.key;
+const emmRiskFactorsKeySelector = (d: KeyValueElement) => (isTruthyString(d.key) ? d.key : 'None');
+const emmRiskFactorsLabelSelector = (d: EmmEntity) => (isTruthyString(d.label)
+    ? `${d.label} (${d.totalCount})`
+    : `None (${d.totalCount})`
+);
+
+const emmEntitiesKeySelector = (d: EmmEntity) => d.key;
+const emmEntitiesLabelSelector = (d: EmmEntity) => `${d.label} (${d.totalCount})`;
+const emmTriggerKeySelector = (d: EmmEntity) => d.key;
+const emmTriggerLabelSelector = (d: EmmEntity) => `${d.label} (${d.totalCount})`;
+
+function FilterForm(props: OwnProps) {
+    const {
+        className,
+        projectId,
+        filterValues,
+        entriesFilters,
+        entriesWidgets,
+        onChange,
+        filterOnlyUnprotected,
+        regions,
+        geoOptions,
+        hasAssessment,
+    } = props;
+
+    const schema = useMemo(() => {
+        const leadSchema = {
+            search: [],
+            assignee: [],
+            created_at: [],
+            published_on: [],
+            confidentiality: [],
+            status: [],
+            priority: [],
+            authoring_organization_types: [],
+            exists: [],
+            emm_risk_factors: [],
+            emm_keywords: [],
+            emm_entities: [],
+        };
+        if (hasAssessment) {
+            return {
+                fields: {
+                    ...leadSchema,
+                },
+            };
+        }
+        return {
+            fields: {
+                ...leadSchema,
+                entries_filter: {
+                    fields: {
+                        created_by: [],
+                        comment_assignee: [],
+                        comment_created_by: [],
+                        comment_status: [],
+                        verified: [],
+                        entry_type: [],
+                        project_entry_labels: [],
+                        lead_group_label: [],
+                        ...listToMap(entriesFilters, v => v.key, v => []),
+                    },
+                },
+            },
+        };
+    }, [entriesFilters, hasAssessment]);
+
+    const [faramValues, setFaramValues] = useState<FaramValues>(filterValues);
+    const [faramErrors, setFaramErrors] = useState<FaramErrors>({});
+    const [pristine, setPristine] = useState(true);
+
+    const filteredFrameworkFilters = useMemo(() => {
+        const widgetsMap = listToMap(entriesWidgets, d => d.key, d => d.widgetId);
+        const filtersWithId = entriesFilters?.map(f => ({
+            ...f,
+            widgetId: widgetsMap[f.widgetKey],
+        }));
+        return filtersWithId ? [...filtersWithId] : [];
```

Comment:
why make a copy of filtersWithId?
```suggestion
        return filtersWithId ?? [];
```